base_layer/wallet/migrations/2021-03-23-082938_update-outputs-for-tari-script/up.sql

```diff
@@ -1,12 +1,23 @@
 -- This migration is part of a testnet reset and should not be used on db's with existing old data in them
 -- thus this migration does not accommodate db's with existing rows.
-ALTER TABLE outputs
-    ADD COLUMN script BLOB NOT NULL;
-ALTER TABLE outputs
-    ADD COLUMN input_data BLOB NOT NULL;
-ALTER TABLE outputs
-    ADD COLUMN height INTEGER NOT NULL;
-ALTER TABLE outputs
-    ADD COLUMN script_private_key BLOB NOT NULL;
-ALTER TABLE outputs
-    ADD COLUMN script_offset_public_key BLOB NOT NULL;
\ No newline at end of file
+
+PRAGMA foreign_keys=off;
+DROP TABLE outputs;
```

Comment:
I find it very unlikely that Mac has a less strict sqlite3 binary than windows and linux? Are you sure you were not just running this migration in a location with an existing db?

Reply:
Very certain there were no old databases present; tested this on three platforms as well.

Reply:
Wow, weird that it works on mac. Well in that case this approach is fine.